src/components/MainApp.tsx

```diff
@@ -14,13 +14,6 @@ import { useAppSelector } from "../store/store";
 
 const { Footer, Content } = Layout;
 
-const ContentStyled = styled(Content)`
-    padding: 0 15px;
-
-    @media (min-width: 768px) {
```

Comment:
Maybe you still need this styled component to make sure only the home page is affected by the Desktop Version changes?